2024/waypoints/lap_waypoints_task_1.csv

```diff
@@ -1,5 +1,7 @@
 name,latitude,longitude
-1,48.5112151,-71.6508739
-2,48.5055945,-71.6320082
-3,48.5058977,-71.6317428
-4,48.5114552,-71.6506902
\ No newline at end of file
+1,48.505808,-71.632258
+2,48.505842,-71.632111
+3,48.505926,-71.632165
+4,48.511363,-71.650462
+5,48.511312,-71.650668
+6,48.511191,-71.650611
```

Comment:
Nit: I think waypoint 6 and 7 isn't covering Alpha. ![image](https://github.com/UWARG/pathing/assets/55414757/6a038afa-cd6d-4a8c-ba5d-824c476033a0)